blog/2022-11-22 featureprobe-md

```diff
@@ -0,0 +1,43 @@
+
+slug | title
+-- | --
+FeatureProbe Rust | 用 Rust 开发跨平台 SDK 探索和实践
```

Comment:
Is this title right?

Reply:
Not at all, I copied the last article......